apps/storybook/src/Selection.stories.tsx

```diff
@@ -106,6 +116,32 @@ ChangeSelectionColors.argTypes = {
   },
 };
 
+export const SelectToZoom: Story<{
```

Comment:
Please try the new story yourself and tell me how it feels !

Reply:
Perhaps this story should live in its own file, with title `Building Blocks/ZoomSelectionMesh`.